core/toolbox.js

```diff
@@ -650,7 +652,7 @@ Blockly.Toolbox.prototype.updateColourFromTheme_ = function(opt_tree) {
  * Updates the category colours and background colour of selected categories.
  * @package
  */
-Blockly.Toolbox.prototype.updateColourFromTheme = function() {
+Blockly.Toolbox.prototype.applyTheme = function() {
```

Comment:
There's precedence for this in workspace_svg. Perhaps we should keep the name consistent with that: 
``Blockly.WorkspaceSvg.prototype.refreshTheme``.